lib/src/linden/xsizes.dart

```diff
@@ -42,6 +42,7 @@ class XSizes {
   final double unit6 = _unit * 6;
   final double unit7 = _unit * 7;
   final double unit8 = _unit * 8;
+  final double unit8_25 = _unit * 8.28;
```

Comment:
❓ Is the value 8.28 correct or should it be 8.25 ? Just wondering because the name of the variable is 8_25 :)

Reply:
😅It should be 8.25 of course! Thanks for checking